services/queue/src/api.js

```diff
@@ -300,6 +300,132 @@ builder.declare({
   return res.reply(result);
 });
 
+/** Cancel all tasks in a group */
+builder.declare({
+  method: 'post',
+  route: '/task-group/:taskGroupId/cancel',
+  name: 'cancelTaskGroup',
+  scopes: {
+    AnyOf: [
+      'queue:cancel-task-group:<schedulerId>/<taskGroupId>',
```

Comment:
tricky part here is that we call this endpoint with `taskGroupId` parameter, and in order to figure out what `schedulerId` is, we have to fetch first task in the group to construct proper scopes. And since `schedulerId/taskGroupId` is unique, this should be fine